torch/csrc/jit/passes/quantization/quantization_patterns.h

```diff
@@ -1135,9 +1135,13 @@ graph(%a_dequant, %w_quant, %b, %stride, %padding, %dilation, %groups):
         %r = aten::conv1d(%a_dequant, %w_dequant, %b, %stride, %padding, %dilation, %groups)
         return (%r) )";
 
+  // Note: input_qrange_le_128 is always set to True here because this codepath
```

Comment:
This is in regards to graph mode quantization being deprecated?

Reply:
correct